srcpkgs/oneko/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'oneko'
+pkgname=oneko
+version=1.2.5
+revision=1
+# other archs end with /usr/bin/xmkmf: line 53: 15231 Aborted
+archs="x86_64"
```

Comment:
Remove arch restriction and set `nocross=yes` instead.
xvkbd does this too because xmkmf is apparently broken for cross builds.

Reply:
i find that weird because debian packages have oneko for many architectures.
https://packages.debian.org/sid/games/oneko

Reply:
They don't cross compile